src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java

```diff
@@ -96,18 +97,52 @@ public TestCaseView createTestCaseView(
           !ServiceMessages.s_isEmptyType(methodConfig.getLongRunningConfig().getReturnType());
     }
 
+    InitCodeContext responseInitCodeContext =
+        createResponseInitCodeContext(methodContext, initCodeContext.symbolTable());
+    MockGrpcResponseView mockGrpcResponseView =
+        createMockResponseView(methodContext, responseInitCodeContext);
+
+    GrpcStreamingView grpcStreamingView = GrpcStreamingView.newBuilder().build();
```

Comment:
You can default the grpc streaming view to null if it is not a grpc streaming method:
GrpcStreamingView grpcStreamingView;

if (methodConfig.isGrpcStreaming()) {
  grpcStreamingView = GrpcStreamingView.newBuilder().build();
}